i18n/vi.json

```diff
@@ -0,0 +1,282 @@
+{
+  "I18N_ABOUT_PAGE_TITLE": "Về Chúng Tôi - Oppia",
+  "I18N_ACTION_BROWSE_EXPLORATIONS": "Tra cứu các cuộc hành trình",
+  "I18N_ACTION_CREATE_EXPLORATION": "Tạo cuộc hành trình của bạn",
+  "I18N_CREATE_ACTIVITY_QUESTION": "Bạn muốn tạo ra cái gì?",
+  "I18N_CREATE_ACTIVITY_TITLE": "Tạo ra hoạt động mới",
+  "I18N_CREATE_COLLECTION": "Tạo nhóm",
+  "I18N_CREATE_EXPLORATION": "Tạo cuộc hành trình",
+  "I18N_CREATE_EXPLORATION_CREATE": "Sáng Tạo",
+  "I18N_CREATE_EXPLORATION_QUESTION": "Bạn có muốn tạo ra cuộc hành trình mới không?",
+  "I18N_CREATE_EXPLORATION_TITLE": "Tạo một cuộc hành trình mới",
+  "I18N_CREATE_EXPLORATION_UPLOAD": "Đăng Tải",
+  "I18N_CREATE_NEITHER_THANKS": "Không Phải Cả Hai",
+  "I18N_CREATE_NO_THANKS": "Không Phải",
+  "I18N_CREATE_YES_PLEASE": "Đúng Rồi!",
+  "I18N_DASHBOARD_MY_COLLECTIONS": "Bộ Suy Tầm Của Bạn",
+  "I18N_DASHBOARD_MY_EXPLORATIONS": "Các Cuộc Hành Trình Của Bạn",
+  "I18N_DIRECTIVES_DRAG_IMAGE_HERE": "Kéo bức hình tới đây",
+  "I18N_DIRECTIVES_OR_SELECT_FILE": "Duyệt tìm một bức hình trên máy tính của bạn:",
+  "I18N_ERROR_400_PAGE_TITLE": "Lỗi 400 - Yêu cầu này bị lỗi - Oppia",
+  "I18N_ERROR_401_PAGE_TITLE": "Lỗi 401 - Không có quyền - Oppia",
+  "I18N_ERROR_404_PAGE_TITLE": "Lỗi 404 - Không tìm thấy - Oppia",
+  "I18N_ERROR_500_PAGE_TITLE": "Lỗi 500 -  Hệ thống bị lỗi - Oppia",
+  "I18N_ERROR_DISABLED_EXPLORATION": "Vô hiệu hóa cuộc hành trình",
+  "I18N_ERROR_DISABLED_EXPLORATION_MESSAGE": "Xin lỗi bạn, nhưng cuộc hành trình này đang bị khóa. Hãy thử lại sau ít phút nữa.",
+  "I18N_ERROR_DISABLED_EXPLORATION_PAGE_TITLE": "Vô hiệu hóa cuộc hành trình này - Oppia",
+  "I18N_ERROR_MESSAGE_400": "Có những lúc máy tính và con người không hiểu nhau. Đây là 1 trong những lúc đó.",
+  "I18N_ERROR_MESSAGE_401": "Bạn không nên vào đây. Mau trở lại trước khi bị giám thị bắt!",
+  "I18N_ERROR_MESSAGE_404": "Xin lỗi, chúng tôi không tìm thấy trang bạn đang yêu cầu.",
+  "I18N_ERROR_MESSAGE_500": "Lỗi hệ thống.",
```

Comment:
Do you think we should add some humor to these warnings? Users won't then freak out when seeing these errors